plugins/services/src/js/columns/ServicesTableMemColumn.tsx

```diff
@@ -1,19 +1,60 @@
 import * as React from "react";
-import { NumberCell } from "@dcos/ui-kit";
+import { Trans } from "@lingui/macro";
+import { NumberCell, Tooltip } from "@dcos/ui-kit";
 
 import Pod from "../structs/Pod";
 import Service from "../structs/Service";
 import ServiceTree from "../structs/ServiceTree";
 import Units from "#SRC/js/utils/Units";
 
-export const ServiceMem = React.memo(({ resource }: { resource: string }) => (
-  <NumberCell>
-    <span>{Units.formatResource("mem", resource)}</span>
-  </NumberCell>
-));
+export const ServiceMem = React.memo(
+  ({
+    resource,
+    limit
+  }: {
+    resource: string;
+    limit: number | string | null;
+    id: string;
+  }) => {
+    if (limit != null && limit !== 0) {
+      return (
+        <NumberCell>
+          <Tooltip
+            id={`mem{id}`}
+            trigger={`${Units.formatResource(
+              "mem",
+              resource
+            )} / ${Units.formatResource("mem", limit)}`}
+            maxWidth={150}
+          >
+            <Trans
+              id="{resource} are being used with a limit of {limit}"
+              render="span"
+              values={{
+                resource: Units.formatResource("mem", resource),
+                limit: Units.formatResource("mem", limit)
+              }}
+            />
+          </Tooltip>
+        </NumberCell>
+      );
+    }
+    return (
+      <NumberCell>
+        <span>{Units.formatResource("mem", resource)}</span>
+      </NumberCell>
+    );
+  }
+);
 
 export function memRenderer(
   service: Service | Pod | ServiceTree
 ): React.ReactNode {
-  return <ServiceMem resource={service.getResources()[`mem`]} />;
+  return (
+    <ServiceMem
+      id={service.getId()}
+      resource={service.getResources()[`mem`]}
```

Comment:
can we simplify this?

``service.getResources()[`mem`] `` => `service.getResources().mem`?

Reply:
✅